### ISSUE TITLE
availability table, attempt to book and center booked not matched

### ISSUE THREAD

**Issue by Kanchangk** [bombardier-gif/covid-vaccine-booking#330]
![image](https://user-images.githubusercontent.com/8496126/119438757-8744ff00-bd3e-11eb-867d-624c56726aab.png)

Showing attempt to book center is at index 0, but actual booked center was different and of different date. 

It is happening due to randomization of slots after displaying the table. But 
1. ironically, it did not tried for the center with highest availability.
2. When the response status shows that center is already full, it still keeps on trying to book slots (may be coz API data is not updated.)

**Comment by malayilneil** [comment]
A similar Issue happened to me yesterday. A Hospital released all their slots for the rest of the week at once. Bot tried to book the slot but returned it was completely booked. Instead of trying to book other slots on other dates it tried the same thing again.  

Issues faced
1) Similar to the table shown above, the program showed on the second and third try that 100 slots (100 slots were released by the hospital for that date) were still available while in reality it was completely booked since that's what it returned on the first try. 
2) It tried the same date again and again instead of looking for updated alternative dates with most no of slots available. 

@bombardier-gif  Might be a tough ask, but anyways to alleviate this issue? I used the most updated code and just started the program 20 seconds before releasing the slots (to prevent caching).
